.npmrc

```diff
@@ -1,2 +1 @@
-save-exact = true
-
+save-prefix = ^
```

Comment:
I believe this is the default. We could probable just delete the file for the same result.